scripts/prepare_dutchf3.py

```diff
@@ -253,48 +328,89 @@ def split_alaudah_et_al_19(data_dir, stride, patch_size, fraction_validation=0.2
 
     list_train_val = i_list + x_list
 
-    # create train and test splits:
-    train_list, test_list = train_test_split(list_train_val, test_size=fraction_validation, shuffle=True)
+    # create train and validation splits:
+    train_list, val_list = train_val_split(list_train_val, val_size=fraction_validation, shuffle=True)
```

Comment:
this is a function imported from scikit-learn. it should be `train_test_split` not `train_val_split`.. otherwise this would fail.

Reply:
same goes for `val_size`.. it should be `test_size`

Reply:
I didn't catch this as an error because it is inside the split_alaudah_et_al_19 method. But I returned it to the original value.